storage/storage-interface/src/state_view.rs

```diff
@@ -65,3 +71,33 @@ impl DbStateViewAtVersion for Arc<dyn DbReader> {
         })
     }
 }
+
+impl ModuleViewer for DbStateView {
```

Comment:
Can we implement ModuleViewer for TStateView directly (at StateView definition site) or something of the sort, which would also allow us to re-use the implementation with DebuggerStateView?

Reply:
We can only do impl for some S: StateView if in the same crate. I am not sure what is the best workaround here?

Reply:
No, ModuleViewer lives in move repo, and there we do not have state view :(